src/Tasks/CheckComposerUpdatesTask.php

```diff
@@ -340,25 +340,24 @@ protected function isStableEnough($currentStability, $possibleStability)
     protected function recordUpdate($package, $installed, $latest)
     {
         // Is there a record already for the package? If so find it.
-        $packages = ComposerUpdate::get()->filter(['Name' => $package]);
+        $packages = Package::get()->filter(['Name' => $package]);
+
+        // Get the hash installed
+        $localPackage = $this->getLocalPackage($package);
+        $installedHash = $localPackage->source->reference;
 
         // if there is already one use it otherwise create a new data object
         if ($packages->count() > 0) {
             $update = $packages->first();
         } else {
-            $update = ComposerUpdate::create();
+            $update = Package::create();
             $update->Name = $package;
-        }
-
-        // If installed is dev-master get the hash
-        if ($installed === 'dev-master') {
-            $localPackage = $this->getLocalPackage($package);
-            $installed = $localPackage->source->reference;
+            $update->Version = $installed;
```

Comment:
Will also need to set `Description` and `Type` if this is to replace the default refresh task included with `silverstripe-maintenance`

Reply:
It’s not aimed at that, expect more follow ups to come

Reply:
In this PR, or just in general?
(ie. merge now?)

Reply:
Further PRs (there have been a few building up)

Reply:
Sweet I'll merge then :)